bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/ui/launch/shortcuts/StartAction.java

```diff
@@ -99,42 +98,16 @@ public void launch(IEditorPart part, String mode) {
      * 
      * @throws Exception
      */
-    public static void run(IProject iProject, ILaunchConfiguration iConfiguration, String mode) throws Exception {
-
-        if (Trace.isEnabled()) {
-            Trace.getTracer().traceEntry(Trace.TRACE_UI, new Object[] { iProject, iConfiguration, mode });
-        }
-
-        if (iProject == null) {
-            throw new Exception("Invalid project. Be sure to select a project first.");
-        }
+    public static void run(IProject iProject, String mode) throws Exception {
```

Comment:
Removing the tracing here made me wonder if we have enough on the overall flow.  Looking at io.openliberty.tools.eclipse.ui.launch.LaunchConfigurationHelper.getLaunchConfiguration(IProject, String, RuntimeEnv)  I'm not sure we do.  I wonder if we should add some tracing to that method?